classes/c++.oeclass

```diff
@@ -38,13 +38,13 @@ SDK_CXXFLAGS		?= "${SDK_CFLAGS} -fpermissive"
 CXXFLAGS			 = "${HOST_CXXFLAGS}"
 HOST_CXXFLAGS:native		 = "${BUILD_CXXFLAGS}"
 HOST_CXXFLAGS:cross		 = "${BUILD_CXXFLAGS}"
-HOST_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS}"
+HOST_CXXFLAGS:machine		 = "${MACHINE_CXXFLAGS} ${PIC_OPTION}"
```

Comment:
Oh, you are looking at TARGET_CPU_powerpc to control PIC_OPTION, so adding to HOST_CXXFLAGS does not look right.